exporter/awss3exporter/marshaler_test.go

```diff
@@ -18,6 +18,12 @@ func TestMarshaler(t *testing.T) {
 		require.NotNil(t, m)
 		assert.Equal(t, m.format(), "json")
 	}
+	{
+		m, err := newMarshaler("otlp_proto", zap.NewNop())
+		assert.NoError(t, err)
+		require.NotNil(t, m)
+		assert.Equal(t, m.format(), "binpb")
```

Comment:
Is `binpb` a good term here?

Reply:
I did have to look up what a binary protocol buffer file extension should be, according to the Protocol Buf site:
https://protobuf.dev/programming-guides/techniques/#suffixes

Reply:
Nice, thanks for the link! :)